crates/bevy_ui/src/render/ui.wgsl

```diff
@@ -39,4 +39,4 @@ fn fragment(in: VertexOutput) -> @location(0) vec4<f32> {
         color = in.color;
     }
     return color;
-}
+}
```

Comment:
This change should be removed. :)

Reply:
I might have made a complete fool out of myself here :P 
but is this a windows line ending or was that just a joke that completely went past myself?